modules/api/models.py

```diff
@@ -78,15 +79,16 @@ def merge_class_params(class_):
                 field=underscore(fields["key"]), 
                 field_alias=fields["key"], 
                 field_type=fields["type"],
-                field_value=fields["default"]))
+                field_value=fields["default"],
+                field_exclude=fields["exclude"] if "exclude" in fields else False))
```

Comment:
Allows us to exclude fields from being passed into the processing model

Reply:
@ArcticFaded you might also be interested

Reply:
Yeah i have no problem with excludes